addons/forum-post-countdown/addon.json

```diff
@@ -21,6 +21,15 @@
       "link": "https://scratch.mit.edu/users/mybearworld/"
     }
   ],
+  "settings": [
+    {
+      "name": "Retry after countdown",
+      "description": "When posting fails because of the 60 second rule, automatically try again when possible.",
+      "type": "boolean",
+      "default": true,
```

Comment:
@mxmou @Samq64 Is it OK if we change this to false?

Reply:
Yes, false is probably a better default.

Reply:
OK #8282